patch_parser_test.py

```diff
@@ -1,19 +1,38 @@
 import unittest
 from patch_parser import parse_comments
-from archive_converter import generate_email_from_file
+from archive_converter import find_thread, generate_email_from_file
 
-class Patch_Parser_Test(unittest.TestCase):
+class PatchParserTest(unittest.TestCase):
 
     def test_parse_comments_for_single_email_thread(self):
-        email = generate_email_from_file("test_data/patch6.txt")
+        email = find_thread('PATCH v2 1/2')
         patchset = parse_comments(email)
         self.assertTrue(len(patchset.patches) > 0)
-        self.assertTrue(patchset.patches[0].set_index == 0)
+        self.assertEqual(patchset.patches[0].set_index, 0)
         self.assertTrue(len(patchset.patches[0].text) > 0)
-        self.assertTrue(len(patchset.patches[0].text_with_headers) > 18)
+        self.assertGreater(len(patchset.patches[0].text_with_headers),18)
         self.assertEqual(patchset.patches[0].comments, [])
         
+<<<<<<< HEAD
     #TODO(willliu@google.com): Add tests for Multiple patches, no cover letter, Multiple patches with cover letter, Email no patches.
+=======
+    def test_parse_comments_for_multi_email_thread_with_cover_letter(self):
+        email = find_thread('PATCH v2 0/4')
```

Comment:
I'd put a line break between this an the asserts.

And perhaps do

```python
  patchset = parse_comments(find_thread('...')

  self.assert...
```

as we don't use `email` anywhere else.